mix.exs

```diff
@@ -83,7 +83,9 @@ defmodule Honeybadger.Mixfile do
 
   defp deps do
     [
-      {:hackney, "~> 1.1"},
+      {:finch, "~> 0.19", optional: true},
```

Comment:
Since we want to nudge people toward `req`, and I suspect _very_ few applications would use `finch` directly without `req`, I'd vote to omit a dedicated `finch` adapter entirely.

Reply:
This is a good observation, I'll remove that adapter.